backend/cli.py

```diff
@@ -71,8 +71,12 @@ def add_imdb_ratings():
     typer.echo("Adding ratings to meilisearch...")
 
     imdb_media = [
-        {"id": media.id, "imdb_rating": float(imdb_ratings[media.imdb_id])}
+        {
+            "id": media.id,
+            "imdb_rating": float(imdb_ratings[media.imdb_id].get("rating")),
+        }
         if imdb_ratings.get(media.imdb_id)
+        and int(imdb_ratings[media.imdb_id].get("votes")) >= min_votes
```

Comment:
Is there a reason we are mixing `[]`'s and `get()`'s?

Reply:
not really no - changed it